### PR TITLE
Footer

### PR DESCRIPTION
**Pull request by PNQT**

### CODE REVIEW

File: src/layout/footer.html [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+            <div class="column_name_list">
+                <a class="column_name_list_link"  href="#">
+                     Career
+                </a>
+            </div>
+            <div class="column_name_list">
+                <a class="column_name_list_link" href="#">
+                     Mobile
+                </a>
+            </div>
```
Comment (PhamNhuQuocBao) [inline src/layout/footer.html:46]: should create component fot this.

File: src/layout/footer.html [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+            <div class="column_name_list">
+                <a class="column_name_list_link" href="#">
+                     FAQ's
+                </a>
+            </div>
+            <div class="column_name_list">
+                <a class="column_name_list_link" href="#">
+                     Blog
+                </a>
+            </div>
```
Comment (PhamNhuQuocBao) [inline src/layout/footer.html:73]: same above comment, u can reuse because they are same component.

File: src/stylesheet/layout/_footer.scss
Comment (PhamNhuQuocBao) [inline src/stylesheet/layout/_footer.scss:1]: format code, pls.

**Review by PhamNhuQuocBao — APPROVED** [review]

File: src/layout/footer.html [hunk 3]
```diff
@@ -29,67 +29,73 @@ <h3 class="column_name">
...
+                <div class="column_name_list">
+                    <a class="column_name_list_link" href="#">
+                         FAQ's
+                    </a>
+                </div>
+                <div class="column_name_list">
+                    <a class="column_name_list_link" href="#">
+                         Blog
+                    </a>
+                </div>
```
Comment (PhamNhuQuocBao) [inline src/layout/footer.html:76]: ?

**Review by PhamNhuQuocBao — CHANGES_REQUESTED** [review]
resolve conflicts

File: src/layout/footer.html (hunk 3 above)
Comment (PNQT) [reply src/layout/footer.html:76]: I don't know how to create components by html for it

Comment (datmaiz) [reply src/layout/footer.html:76]: image
split to component like this and reuse like this:
image

Comment (PNQT) [reply src/layout/footer.html:76]: Thank you. I will try it after . when i finished all responsive

**Pull request merged**